### DISCUSSION TITLE
Algolia driver: How to hide unpublished pages from search results

### DISCUSSION THREAD

**Discussion by markussorg** [statamic/cms#5200] [Q&A] [answered]
Hey,

I'm using Algolia as my search driver and Algolia returns "unpublished" pages in my search results (leads to 404s). Does anybody know how to prevent that? I only want to show "published" entries in the results. 

Can I control on the statamic side which entries gets indexed (based on the status)? Or do I have to filter out "unpublished" entries on the algolia side? If so, does anybody know how to do this?

Thanks in advance...

**Comment by riasvdv** [comment, marked as answer]
I add the `published` field to the search index, and configure Algolia to have a `AND published:true` as a default filter using the https://www.algolia.com/doc/api-reference/widgets/configure/js/ widget
